components/nordigen/nordigen.app.mjs

```diff
@@ -6,8 +6,134 @@ export default {
   propDefinitions: {
     country_code: {
       type: "string",
-      label: "Enter a country code",
-      description: "[ISO 3166](https://www.iso.org/obp/ui/en/#iso:pub:PUB500001:en) two-character code for the country (eg. `FR`) for one of [these supported countries](https://nordigen.com/en/coverage/).",
+      label: "Country",
+      description: "Country where your bank is located.",
+	  options: [
```

Comment:
Can you please move these options to a common file like `nordigen/common/constants.mjs` like in [this app file](https://github.com/PipedreamHQ/pipedream/blob/1e25f764834e448c49d10a513a295650a2a2828f/components/apitemplate_io/app/apitemplate_io.app.ts#L3) and [this constants file](https://github.com/PipedreamHQ/pipedream/blob/master/components/apitemplate_io/common/constants.ts)? This way app file might be too big.

Reply:
Done